tests/20-topdown.py

```diff
@@ -2,11 +2,17 @@
 
 import amulet
 import os
-import subprocess
-import time
 import unittest
 import yaml
 
+from amulet_utils import (
```

Comment:
I want to ding this on style points ;) but i'm fine with this as is and I'll expand the imports